docs/index.asciidoc

```diff
@@ -223,16 +223,16 @@ Note that 409 exceptions are no longer retried. Please set a higher `retry_on_co
 It is more performant for Elasticsearch to retry these exceptions than this plugin.
 
 [id="plugins-{type}s-{plugin}-dlq-policy"]
-==== DLQ Policy
+==== DLQ policy
 
 Mapping (404) errors from Elasticsearch can lead to data loss. Unfortunately
 mapping errors cannot be handled without human intervention and without looking
 at the field that caused the mapping mismatch. If the DLQ is enabled, the
 original events causing the mapping errors are stored in a file that can be
 processed at a later time. Often times, the offending field can be removed and
 re-indexed to Elasticsearch. If the DLQ is not enabled, and a mapping error
-happens, the problem is logged as a warning, and the event is dropped. See
-<<dead-letter-queues>> for more information about processing events in the DLQ.
+happens, the problem is logged as a warning, and the event is dropped. 
+Check out https://www.elastic.co/guide/en/logstash/8.18/dead-letter-queues.html[Dead letter queues (DLQ)] for more information about processing events in the DLQ.
```

Comment:
🤔 Is 8.18 intentional change here? If not maybe we change it with `logstash-ref`?

```suggestion
Check out {logstash-ref}/dead-letter-queues.html[Dead letter queues (DLQ)] for more information about processing events in the DLQ.
```